backend/src/DatabaseHandler.py

```diff
@@ -62,19 +63,27 @@ def send_message_to_database(json_data: str):
         """
         # Extract the table name from the JSON data
         json_data = json.loads(json_data)
+        if len(list(json_data.keys())) < 3:
+            logger.warning(f"Received, poorly formed json: {json_data}")
+            return
+
         table_name = list(json_data.keys())[2]
 
         logger.info(f"Adding an entry to the {table_name} table")
-        logger.trace(f"Entry: {json_data[table_name]}")
+        logger.info(f"Entry: {json_data[table_name]}")
 
         # Push the JSON data to PocketBase using the correct schema
-        DatabaseHandler.client.collection(table_name).create(json_data[table_name])
+        try:
+            DatabaseHandler.client.collection(table_name).create(json_data[table_name])
+        except Exception:
+            logger.error(f"Failed to create entry in {table_name}: {json_data}")
 
 # Procedures ======================================================================================
 def database_thread(thread_name: str, db_workq: mp.Queue, message_handler_workq: mp.Queue):
     """
     The main loop of the database handler. It subscribes to the CommandMessage collection
     """
+    logger.info(f"Database process: {os.getpid()}")
```

Comment:
Probably want to get rid of this after the core usage has been debugged.

Reply:
yeah that is true for now ill leave it until we get another change to debug but ill put a comment by them as well